src/SprykerSdk/Shared/AsyncApi/Transfer/asyncapi.transfer.xml

```diff
@@ -1,67 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
This file must not be removed, please only rename it so the TransferFinder can't find this file anymore.

`src/SprykerSdk/Transfer/asyncapi.xml`